docs/Reference/Rest.md

```diff
@@ -22,6 +22,33 @@ The default location is `http://localhost:9000/swagger-ui`.
 
 You can also use tools such as [Postman] or [cURL] to interact with Web3Signer APIs.
 
+## Endpoint example usage
+
+### `/api/v1/eth2/publicKeys`
+
+Returns a hex-encoded list of ETH2 BLS public keys for the private keys that have been loaded into Web3Signer.
+
+### `/api/v1/eth2/sign/{identifier}`
+
```

Comment:
The data to be signed needs to be specified in the request body. Giving an example for this might be bit tricky there are a lot possibilities.